src/test/shared/sam/cli/samCliTestUtils.ts

```diff
@@ -26,3 +27,36 @@ export class MockSamCliProcessInvoker implements SamCliProcessInvoker {
         } as any as ChildProcessResult
     }
 }
+
+export function assertArgsContainArgument(
+    args: any[],
+    argOfInterest: string,
+    expectedArgValue: string
+) {
+    const argPos = args.indexOf(argOfInterest)
+    assert.notStrictEqual(argPos, -1, `Expected arg ${argOfInterest} was not found`)
```

Comment:
Nit: Throughout these messages what do you think about wrapping `argOfInterest` in quotes, i.e. `'${argOfInterest}'`?

Reply:
Good utility in that, especially because we're using these methods a fair bit